packages/site/pages/components/position-positions-example.js

```diff
@@ -4,10 +4,27 @@ import core from '@pluralsight/ps-design-system-core'
 import Button from '@pluralsight/ps-design-system-button'
 import Tooltip from '@pluralsight/ps-design-system-tooltip/react.js'
 
-import * as components from '@pluralsight/ps-design-system-position/react.js'
+import {
+  Above,
+  AboveLeft,
+  AboveRight,
+  BelowLeft,
+  BelowRight,
+  RightOf,
+  LeftOf
+} from '@pluralsight/ps-design-system-position'
 
 import { Head } from '../../src/ui/index.js'
 
+const components = [
```

Comment:
Yea I talked to Dane about this Friday. We both agreed most react devs know `Components` have to start capitalized so it makes sense the utility function that matches it would camelCased. This is the only react package that doesn't have a default export I believe, have to check to be sure.  

Still I also see your point which is why I talked to Dane about it. I'll think about this over lunch and and maybe reach back out once I solve this test bug.